files/SZWESAMP/ZWERSTC

```diff
@@ -0,0 +1,34 @@
+//ZWERSTC JOB                                     
+//*                                                                    
+//* This program and the accompanying materials are made available     
+//* under the terms of the Eclipse Public License v2.0 which          
+//* accompanies this distribution, and is available at                 
+//* https://www.eclipse.org/legal/epl-v20.html                         
+//*                                                                    
+//* SPDX-License-Identifier: EPL-2.0                                   
+//*                                                                    
+//* Copyright Contributors to the Zowe Project. 2020, 2020             
+//*                                                                    
+//********************************************************************* 
+//*
+//* This job is used to remove proclib members
+//* Used to start a Zowe "instance"
+//* Instances represent a configuration of Zowe, different from the
+//* "runtime" datasets that are created upon install of Zowe / SMPE.
+//*
+//********************************************************************* 
+//*
+//*      * You do NOT need to change PROCLIB when running ZWEGENER.
+//*      * This is used to keep some lines under the column limit.
+//*
+//       SET PROCLIB={zowe.setup.dataset.proclib}
```

Comment:
Missing `EXPORT SYMLIST=*` and `SYMBOLS=JCLONLY` for `//SYSTSIN  DD *`

Reply:
[Done.](https://github.com/zowe/zowe-install-packaging/pull/3718/commits/08c398a24379b00402235c9d721208bf0f034e17)